bundles/core/org.eclipse.smarthome.core.audio.test/.settings/org.eclipse.jdt.groovy.core.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+groovy.compiler.level=24
```

Comment:
Shouldn't we use -1 as version, so the workspace one is used and not a different one if we ever want to change that.
We already fixed a bunch of references after the 23 to 24 bump.